css/styles.css

```diff
@@ -1392,6 +1420,9 @@ img {
 }
 .footer-nav {
   display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
```

Comment:
You don't need all these 3 property:

1. `flex-direction: row;` is default value;
2.` flex-wrap: wrap;` no need because all of your elements in one row
3. `justify-content: center; ` moves your headings (Company and Subscribe) to the center when they should be on the left:
<img width="590" alt="Screenshot 2024-09-05 at 12 45 13" src="https://github.com/user-attachments/assets/4848c24d-ea42-4af2-b652-a3d78a232e1f">
<img width="992" alt="Screenshot 2024-09-05 at 12 45 40" src="https://github.com/user-attachments/assets/7ea04479-1633-4244-a07c-4230198ffd78">